core/src/main/scala/org/broadinstitute/dsde/rawls/util/HttpClientUtils.scala

```diff
@@ -34,7 +34,7 @@ trait HttpClientUtils extends LazyLogging {
     executeRequest(http, httpRequest) recover { case t: Throwable =>
       throw new RawlsExceptionWithErrorReport(
         ErrorReport(StatusCodes.InternalServerError,
-                    s"HTTP call failed: ${httpRequest.uri}. Response: ${t.getMessage}",
+                    s"HTTP call failed: ${filterPrivate(httpRequest.uri)}. Response: ${t.getMessage}",
```

Comment:
It felt helpful to include the target uri in the error message, as long as that uri was not private; so I added a filter instead of removing the uri altogether